src/Nirum/Targets/Python.hs

```diff
@@ -1069,6 +1072,22 @@ class #{className}(#{T.intercalate "," $ compileExtendClasses annotations}):
 
     __nirum_type__ = 'union'
 
+    __nirum_annotations__ = map_type({
+%{ forall (annoIdentifier, annoArgSet) <- M.toList $ A.annotations annotations }
+        '#{toAttributeName annoIdentifier}': map_type({
+%{ forall (annoArgIdent, annoArg) <- M.toList annoArgSet }
+            '#{toAttributeName annoArgIdent}':
+%{ case annoArg }
+%{ of AI.Text t }
+                u'''#{escapeSingle t}''',
```

Comment:
Do we need the `escapeSingle` function?  I believe there's already a function defined for render `Text` as a Python string literal (including opening and closing quotes). /cc @kanghyojun